plugins/miscellanea/snapcast/default/snapclient

```diff
@@ -0,0 +1,22 @@
+# defaults file for snapclient
+
+# start snapclient automatically?
+START_SNAPCLIENT=true
+
+# Allowed options:
+#   --help                          produce help message
+#   -v, --version                   show version number
+#   -l, --list                      list pcm devices
+#   -s, --soundcard arg (=default)  index or name of the soundcard
+#   -e, --mstderr                   send metadata to stderr
+#   -h, --host arg                  server hostname or ip address
+#   -p, --port arg (=1704)          server port
+#   -d, --daemon [=arg(=-3)]        daemonize, optional process priority [-20..19]
+#   --user arg                      the user[:group] to run snapclient as when daemonized
+#   --latency arg (=0)              latency of the soundcard
+#   -i, --instance arg (=1)         instance id
+#   --hostID arg                    unique host id
+
+USER_OPTS="--user snapclient:audio"
+
+SNAPCLIENT_OPTS="-d -h 127.0.0.1 -s sndrpihifiberry"
```

Comment:
Can we replace `sndrpihifiberry` with `sndsnapclient` or something else not tied to a specific DAC vendor?